R/core.R

```diff
@@ -95,9 +96,15 @@ benchmark_run <- function(expr_before_benchmark =
     abort_string()
   }
 
+  if (rlang::is_interactive()) {
+    n <- Sys.getenv("touchstone.n_runs", unset = NA) %>% as.numeric() %|% n
```

Comment:
I think to follow the *repeat yourself* paradigm in documentation, we should emit a message here that n is overridden and also in the documentation above for `n`, we could make this explicit,